builtin/settingtypes.txt

```diff
@@ -151,6 +151,16 @@ enable_hotbar_mouse_wheel (Hotbar: Enable mouse wheel for selection) bool true
 #    Requires: keyboard_mouse
 invert_hotbar_mouse_wheel (Hotbar: Invert mouse wheel direction) bool false
 
+#    SDL only: Save key controls as scancodes (position on the keyboard) instead of keycodes (the
```

Comment:
we aim to default to sdl, so IMO we should ignore the "non-sdl" case in descriptions and such

Reply:
I removed non-SDL-related explanations in d34dc3520f03af854948ad51d3b609a73c2eb250.